src/dev-spaces-preview/azext_dev_spaces_preview/_help.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+
+helps['aks use-dev-spaces'] = """
+    type: command
+    short-summary: (PREVIEW) Use Azure Dev Spaces with a managed Kubernetes cluster.
+    parameters:
+        - name: --name -n
+          type: string
+          short-summary: Name of the managed cluster.
+        - name: --resource-group -g
+          type: string
+          short-summary: Name of resource group. You can configure the default group. Using `az configure –defaults group=<name>`.
```

Comment:
I'm not sure the CLI automatically wraps and aligns help text - can you check? There is supposed to be a line break between "group" and "using". Also there shouldn't be a period after "default group." Finally "Using" shouldn't be capitalized.